### ISSUE TITLE
Using Resque in production - Active status of project

### ISSUE THREAD

**Issue by elpd** [resque/resque#1305]
Hello,
I am investigating background queue processing for production and I have seen Resque but the project seems a little stale. I know this is an open source project. It would be helpful if someone can answer and to know is version 2 in active development ? Is version 1 is being maintained actively? Would you recommend it in this time ?

**Comment by mkonikowski** [comment]
:+1:

**Comment by nvquanghuy** [comment]
Me too. Would love to know this to decide on using Resque for a real-world project.

**Comment by utilum** [comment]
After > 40 days it looks like no answer is an answer.
Is there a need for version 1-x-stable to be maintained? For version 2 to be developped?

**Comment by cade** [comment]
https://github.com/resque/resque/issues/1170#issuecomment-38838524

**Comment by utilum** [comment]
I understand this to mean: unless some serious new contribution effort begins, development, as well as maintenance have come to an end. Perhaps the need is not that great. That so?

**Comment by lukeasrodgers** [comment]
FWIW we've been using resque 1.x (currently 1.25.2) in production for several years and it's pretty solid. One issue to be aware of: the web interface on the most recent versions of 1.x branch has some serious problems (e.g. ordering of failed jobs is highly counter-intuitive, you should never use the "retry all/remove all failure" buttons).

I don't have any experience with resque 2.

**Comment by ioquatix** [comment]
+1 what is going on with resque 2.0

**Comment by steveklabnik** [comment]
Here's what's going on:

1. Resque 2.0 is cancelled.
2. I'm maintaining 1.x, with the idea of focusing on compatibility.
3. I don't have a lot of help, so it's going very slow. I have maybe half a day every two weeks or so to put into Resque.

Resque 1.x is a super stable platform used for years by many big companies for lots of things. Sometimes, software doesn't need to change a whole lot to be good. So the only way the slow development _really_ affects you is if you need a particular bug fixed.

I apologize for my silence on this issue, I had taken a long break.

**Comment by ioquatix** [comment]
What does cancelled mean? What's wrong with 2.0?

**Comment by steveklabnik** [comment]
2.0 was originally a ground-up re-write. A lot of work was done, and the basics were in place.

Then, I changed jobs, and was no longer able to work on Resque as much. Life happened to everyone else too. With nobody really left to do the work, releasing a 2.0 would be irresponsible, so we decided to just cancel the release, and focus on 1.x.

I have a very limited amount of time, but some new volunteers have stepped up too. I'm hoping to put out a new 1.x release within a month, depending on how it goes.
